about.html

```diff
@@ -0,0 +1,134 @@
+                        Our website is the best way to find out about our programs and book your stay. We’re happy to
+                        help you plan your trip, whether it’s a weekend away with friends or an adventure holiday with
+                        family.
+                    </p>
+                    <p>Camping is a great way to enjoy the outdoors. With our activities, you can stay fit and have fun
+                        while enjoying your time with friends and family.</p>
+                        <p>
+                            If you love camping, then come and join our friendly community of campers. See what other people are doing on the site and support them!
+                        </p>
+                </div>
+                <div class="sec1-pt3">
+                    <p>Please contact us per Email for any further questions</p>
+                    <p>about All In Campsters</p>
+                    <p>allincampsters@camper.com</p>
+                </div>
+            </div>
+        </section>
+
+        <section id="sec2">
+            <div class="sec2-wrapper">
+                <div class="sec2-pt1">
+                    <h2>All In Campsters</h2>
+                    <h2>Simple and direct logo</h2>
+                    <span>____</span>
+                </div>
+                <div class="sec2-pt2">
+                    <p>The logo of All In Campsters was decided</p>
+                    <p>through the logo competition from 8. June to 7. July</p>
+                    <p>We chose our logo for two main reasons. The first and most important one is that we enjoy working in a logo. The second reason is that we want our users to feel comfortable when they visit our website and interact with all the information on it.</p>
+                </div>
+                <div class="sec2-pt3">
+                    <img src="./assets/All In Campsters.svg" alt="logo">
+                </div>
+            </div>
+        </section>
+
+        <section id="sec3">
+            <div class="sec3-wrapper">
+                <div class="sec3-title">
+                    <h2>See the All In Campsters Blog</h2>
+                    <span>____</span>
+                </div>
+                <div class="sec3-desc">
+                    <p>Take a look at the last two All In Campsters blog posts</p>
+                </div>
+                <div class="sec3-blog">
+                    <span>
+                        <h3 class="blog-title-1"><span>Feb</span><br>Romantic Valentine's Day Destinations</h3>
+                        <img src="./assets/camp-blog-1.jfif" class="blog-img-1" alt="Blog Post 1">
+                    </span>
+                    <span>
+                        <h3 class="blog-title-2"><span>Jan</span><br>Campground Safety: A Guide for campers</h3>
+                        <img src="./assets/camp-blog-2.jpg" class="blog-img-2" alt="Blog Post 2">
+                    </span>
+                </div>
+            </div>
```

Comment:
**OPTIONAL** - Good job so far, but there are some differences between your cards in this section and the Figma design template. Please apply a linear gradient on each card so they colors of the cards look exactly how they are in the Figma template 👍 👍. Remember it is essential that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
<img width="674" alt="sfbsfbs" src="https://user-images.githubusercontent.com/73607512/217219684-f279c1bc-7017-48a3-bb07-8004b91fa93b.PNG">

Figma design:
<img width="557" alt="tsth" src="https://user-images.githubusercontent.com/73607512/217219815-0e194198-3c30-4c6f-8ccf-fa1384bfd3d6.PNG">